web-components/chart/chart.tsx

```diff
@@ -228,8 +268,10 @@ export class CobChart {
 
         // Everytime the selection changes, we want to make sure the the
         // chart still fits nicely on the page and that the chart updates.
-        const selectionName = Object.keys(this.config.selection)[0];
-        const signalName = `${selectionName}_${this.selectField}`;
+        const signalName = this.vegaLite
```

Comment:
Maybe this is fine.

But keep an eye out for ways you could merge things so that it didn't matter if you were using Vega or Vega Lite.

For example, could you make it so that `this.vegaSignal` had the right information regardless of whether it came from Vega or Vega Lite?

Reply:
Worked on this in the most recent commit - added some comments at the top where the global variables are stored. Lmk thoughts.